base/utils/data/trait.dictionary.csv

```diff
@@ -91,4 +91,6 @@
 "SPRTMX";"SPRTMX";"cm";""
 "DMAX";"DMAX";"degree days";""
 "DMIN";"DMIN";"degree days";""
-"leaf_longevity";"leaf longevity";"year";""
\ No newline at end of file
+"leaf_longevity";"leaf longevity";"year";""
+"frprod_perc_10";"fine root allocation percentage vertically in soils (every 10 cm)";"0-100";"frprod_perc_10"
+"gcmax";"Maximum canopy conductance";"m s-1";"gcmax"
```

Comment:
Would be good for this to have the same units as other conductances (kg m-2 s-1)